src/configure/helper/telemetryHelper.ts

```diff
@@ -18,6 +18,9 @@ class TelemetryHelper {
         this.journeyId = uuid();
         this.command = command;
         this.setTelemetry(TelemetryKeys.JourneyId, this.journeyId);
+        if (process.env.DEBUG_MODE === "true") {
+            this.setTelemetry(TelemetryKeys.UserType, "test");
+        }
```

Comment:
this is not required, the sdk already takes care of this by changing the sessionId in the propertybag to this value: 'someValue'

Reply:
![image](https://user-images.githubusercontent.com/20859325/64183340-60b6a480-ce87-11e9-9f3f-5a4ef867f8c7.png)

Reply:
Got it. Thanks. Removed.